[BUGFIX] Correct default icon usage in typeicon_classes (#527)
TYPO3-Documentation/TYPO3CMS-Reference-TCA: Documentation/Ctrl/Properties/TypeiconClasses.rst

   :type: array
   :Scope: Display

   Array of icon identifiers to use for the records. The keys must correspond to the values found in the column
   referenced in the :ref:`typeicon_column <ctrl-reference-typeicon-column>` property. The values correspond

>    to icons registered in the :ref:`Icon API <t3coreapi:icon>`. With the key `default` a default icon is
>    defined which is used when no matching key is found. The default icon is as well used in the
>    "Create new record" dialog from the list module. For using and configuring `typeicon_classes`

Reviewer: ```suggestion
   to icons registered in the :ref:`Icon API <t3coreapi:icon>`. With the key `default`, a default icon is
   defined, which is used when no matching key is found. The default icon is also used in the
   "Create new record" dialog from the List module. For using and configuring `typeicon_classes`
```
